schemas/phenotypic_digest_schema.json

```diff
@@ -1,55 +1,47 @@
 {
     "GLOBAL_COLUMNS": {
         "participant_id": {
-            "Description": "Participant identifier within a given dataset.", 
+            "Description": "Participant identifier.", 
             "dtype": "str",
-            "IsRequired": true,
-            "IsPrefixedColumn": false
+            "IsRequired": true
         },
-        "bids_id": {
-            "Description": "BIDS dataset identifier for a participant, if available/different from the participant_id.", 
+        "bids_participant_id": {
+            "Description": "BIDS-compliant participant identifier.", 
```

Comment:
Same comment as for imaging bagel: maybe we could explicitly say that this should include the `sub-` prefix?